docker/helm/templates/kyuubi-configmap.yaml

```diff
@@ -18,7 +18,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kyuubi-defaults
+  name: {{ .Release.Name }}-kyuubi-defaults
```

Comment:
I would like to keep `{{ .Release.Name }}-kyuubi-defaults` name for now because it represents that `ConfigMap` contains `kyuubi-defaults.conf`.
However, I want to add `kyuubi-env.sh` in next PR , so I'll change name to `{{ .Release.Name }}` similar to other resources.